Components/WebService/Soap/ParseHeadersTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Smartbox\Integration\FrameworkBundle\Components\WebService\Soap;
+
+/**
+ * Trait ParseHeadersTrait.
+ */
+trait ParseHeadersTrait
+{
+    /**
+     * @param $headerContent
+     *
+     * @return array
+     */
+    private function parseHeadersToArray($headerContent)
+    {
+        $headerLines = [];
+        $headers = array_filter(preg_split("(\r\n|\r|\n)", $headerContent));
+
+        foreach ($headers as $line) {
+            if (false === strpos($line, 'POST')) {
```

Comment:
This will not work if method if `!==` than POST. I think it would be a better idea to drop the first row of the `$headers` array

Reply:
Its not only the first line. 
I have added a validation for other HTTP methods